docs/get-details/dapps/smart-contracts/guidelines.md

```diff
@@ -0,0 +1,374 @@
+title: Modern guidelines for smart contracts and smart signatures on Algorand
+
+## Disclaimer
+
+The content provided here is not an exhaustive list of security considerations. Before deploying any application in production, it is strongly advised to get the contract audited by one more third-party security companies.
+
+## DO NOT use smart signatures (except for niche use cases)
```

Comment:
While I agree with this, is this language a bit strong? Maybe something like Use smart signatures sparingly